server/src/main/java/keywhiz/KeywhizConfig.java

```diff
@@ -86,6 +86,13 @@
   @JsonProperty
   private String statusCacheExpiry;
 
+  @JsonProperty
+  private String rowHmacCheck;
+
+  public enum RowHmacCheck {
+    DISABLED, LOGGING, ENABLED
```

Comment:
`DISABLED`, `DISABLED_BUT_LOG`, `ENFORCE` might be better word choices. It's not immediately clear that LOGGING implies it only logs.